index.html

```diff
@@ -424,6 +444,84 @@ <h2 class="experience-text">Skills</h2>
           </li>
         </ul>
       </section>
+
+      <!-- Contact 4/4 - Layout -->
+
+      <section class="l-contact">
+        <!-- Background Images -->
+
+        <div class="e-12"></div>
+        <div class="e-3"></div>
+        <div class="e-5"></div>
+        <div class="r-7"></div>
+        <div class="r-7 r-7-b"></div>
+
+        <!-- Block 1/3 - Text -->
+
+        <div class="l-contact__text">
+          <h2 class="text-title">Contact me</h2>
+          <p class="text-parraf">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+
+        <!-- Block 2/3 - Form -->
+
+        <form action="https://formspree.io/f/mwkaodba" method="post">
+          <ul>
+            <li>
+              <input
+                type="text"
+                class="input"
+                name="name"
+                id="full-name"
```

Comment:
- [ ] **OPTIONAL** Please keep in mind that using a `class` attribute and an `id` attribute on the same element doesn't help at all. You can easily use just one of them (and I recommend the `class` attribute, and that's because the `id` attribute is very unique and shouldn't be used on multiple elements) to refer to the element in the CSS.

Reply:
Good point. Thanks for the tip. I will remember this in the future.

Reply:
Thanks for this suggestion. I understood it and applied it. However, I have found a conflict in this case scenario. 

I use the ids to add the validation functionality through the attribute "aria-describedby="full-name email-address message"" (<---  This are ids) located on the submit button, as in the example below: 
<button
              type="submit"
              id="submit"
              class="c-b-normal-button"
              aria-disabled="true"
              aria-describedby="full-name email-address message"  <-------
            >

I tried to eliminate the class and address the element by its tag name and it didn't apply any style. 
I also tried to apply the style from the scss file by referencing the id name as follows:
#fullname {...} . It didn't work either.

I find myself in the situation where I have to choose to apply the style or keep the validation functionality, that's a dilemma. But if I keep the ids and the class I am able to apply style and also keep the validation working. 
I still take your suggestion as a valuable one and will apply it always that there is not a functionality conflict. However, in this case, and in my humble opinion, I see that it does not apply as expected.
 Please, let me know if there is something that  I can do to implement the changes you suggested and at the same time keep the validation working.
Best regards.